namespaces/live-1.cloud-platform.service.justice.gov.uk/probation-offender-search-dev/00-namespace.yaml

```diff
@@ -10,3 +10,6 @@ metadata:
     cloud-platform.justice.gov.uk/application: "probation-offender-search"
     cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
     cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/offender-search.git"
```

Comment:
should be probation-offender-search and also for dev include probation-offender-search-indexer